articles/introduction-to-sequalize-orm-for-nodejs/index.md

```diff
@@ -0,0 +1,192 @@
+### Introduction to Sequalize ORM for Node.js
+![hero image](/engineering-education/introduction-to-sequalize-orm-for-nodejs/hero.jpg)
+
+### Introduction
+In this article we will have an introductory on SQL database basics, a deep look into Object Relational Mappers, their types, and why one should choose Sequalize over any other existing ORMs with Node.js support, an outlook of what Sequalize is, its basic setup. We will not dive into though into how Sequalize ORM integrates with a Nodejs application. Let us save that for a future tutorial.
+
+### Prerequisites
```

Comment:
Before we talk about installation prerequisites, tell about the conceptual prerequisites. Like what concepts would enable the readers to understand this article better.

Example,
A good understanding about MySQL.